test/scripts/filters/meta_generator.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const should = require('chai').should(); // eslint-disable-line
+
+describe('Meta Generator', function() {
+  const Hexo = require('../../../lib/hexo');
```

Comment:
Now, We are migrating hexo code to ES6 syntax.
Would you please change to arrow function ? Also others.

Reply:
Addressed